docs/guides/quality.md

```diff
@@ -1,142 +1,212 @@
+Re-encodes are encodes of an existing Blu-ray or WEB encode. These are common on [streaming sites](/sourcing/streaming/), where they convert the original encode to a supported file format while also trying to save space.
 
-**Re-encode** - Marked red on nyaa, these are usually encodes of a BDRip, WEB source or even worse. Encoding is a lossy proceess and information is lost at every stage. The same encode made from the BDMV instead of a BDRip would've been better quality. Re-encoding is considered a bad practice.
+The process of re-encoding is generally considered a bad practice due to quality loss, as encoding is a lossy process and information is lost at every stage. As a result, it introduces artifacts like blocking and banding. *Because of this, re-encodes are not recommended if you want quality and should be avoided.*
 
-**Mini encode** - These are around 500mb or lower, they can be either a BDRip or a re-encode. Despite a bad reputation, the minis which are not re-encodes can be a decent option for those low on storage or bandwidth. Mini groups -
+On [Nyaa](https://nyaa.si), these are marked in red. You can see the difference with re-encodes from the comparisons below:
 
-- Good - Judas (new)
-- Decent - Akihitosubs, DB, Ember, Nep_Blanc
-- Bad - Judas (old), Cleo, Reaktor, Cerberus. Their video might be bad, but Cerberus and Reaktor often pick good subtitle sources.
-- Trash - bonkai77, DaddySubs, Tenrai-Sensei, DKB, HR, SSA, FFA, YuiSubs and any other group using NVENC.
+- [Dokyuu Hentai HxEros](https://slow.pics/c/PZRxqAsh) - Raw (Reinforce) vs. Encode (HorribleSubs/Crunchyroll) vs. Re-encode (9anime, animepahe, Gogoanime, Minis)
+- [Oshi no Ko](https://slow.pics/c/6HqApHsn) - Encode (Setsugen, SubsPlease/Crunchyroll) vs. Re-encode (9anime, AniWatch (Zoro), Gogoanime, Marin)
+- [Senran Kagura](https://slow.pics/c/QLtX61qx) - Raw (BDMV, Snow-Raws) vs. Re-encode (9anime, animepahe, AniWatch (Zoro), Gogoanime)
+- [Vinland Saga S2](https://slow.pics/c/GjhwBwo3) - Encode (Foxtrot, SubsPlease/Crunchyroll) vs. Re-encode (Gogoanime, Marin, Minis)
 
-Judas (new) is roughly mid 2020 and later.
-Note that this tier list is valid only for BD sourced encodes and not airing anime. The order of quality for WEB releases is -
++++ Mini Encode
 
-`Subsplease/Erai 1080p > Subsplease/Erai 720p > All mini encodes.`
+Mini encodes are releases designed to save on space and bandwidth while retaining some quality. *These are not the same as re-encodes, as minis encode from the original BD or WEB source.*
 
-### BD vs WEB
+Most mini encodes of Blu-rays are better than a WEB, as they have more data to work with. You can check out some mini-encode comparisons below:
```

Comment:
this no longer applies, so can be removed

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/86/commits/e17de4310044aa9fdd9511c35a0ce45da06b2def.